### PR TITLE
Default isComposed property for custom models 

### PR DESCRIPTION
**Pull request by samvaity**
Default isComposed property on custom models to false if not specified by the service team.

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java [hunk 1]
```diff
@@ -110,7 +111,13 @@ static CustomFormModel toCustomFormModel(Model modelResponse) {
                 PrivateFieldAccessHelper.set(customFormModel, "trainingDocuments",
                     trainingDocumentInfoList);
             }
+        } else {
+            // default to false
+            PrivateFieldAccessHelper.set(customFormModelProperties, "isComposed", false);
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java:116]: python/.net are using `isComposedModel`

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java (hunk 1 above)
Comment (mssfang) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java:116]: The default value of java's primitive type, boolean, is false already.

**Review by mssfang — APPROVED** [review]
Just one comment.

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java (hunk 1 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java:116]: But we weren't setting `customModelProperties` so doing that too :)

**Review by maririos — APPROVED** [review]

**Pull request merged**